pkg/ts/catalog/chart_catalog.go

```diff
@@ -2178,6 +2178,15 @@ var charts = []sectionDescription{
 				},
 				AxisLabel: "Latency",
 			},
+			{
+				Title: "Cancel Requests (Postgres Protocol)",
```

Comment:
maybe "PG Frontend/Backend Protocol"? (that's the name for it on their docs page)

Reply:
hm that seems a bit too verbose IMO